src/components/header.css

```diff
@@ -0,0 +1,63 @@
+.header {
+  width: 100%;
+  height: 100px;
+  position: fixed;
+}
+
+.headerContainer {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 10rem;
+  height: 100px;
+  align-items: center;
+}
+
+.navList {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.linkLi {
+  list-style: none;
+  margin: 0;
+  margin-left: 1rem;
+  letter-spacing: 1.25px;
+  text-transform: uppercase;
+  font-size: 16px;
+}
+
+.linkLi:hover {
+  cursor: pointer;
+}
+
+.navLink {
+  font-family: "Visuelt Bold", sans-serif;
+  color: var(--black);
+  text-decoration: none;
+}
+
+.active {
+  text-decoration: underline;
+}
+
+.backPanel {
+  position: absolute;
+  width: 100%;
+  z-index: -10;
+  transition: all 0.5s ease-out;
+  height: 100px;
+}
+
+.expandedPanel {
+  position: absolute;
+  width: 100%;
+  z-index: -10;
+  height: 50vh;
+  transition: all 0.5s ease-out;
```

Comment:
Let's set this transition up as a variable as well, I suspect we will be reusing it.